### DISCUSSION TITLE
Obscure env vars in console

### DISCUSSION THREAD

**Discussion by ryanblock** [beginner-corp/begin-community#81] [Ideas]
We've heard from a number of folks that they'd like a more secure env var console interface – namely, one that obscures env var values in the web (for example, so that they aren't accidentally shared while screen sharing). We should (and will) definitely add this!

Note: this is a different thing than secrets storage, which is something we're evaluating as a future feature.

**Comment by brianleroux** [comment]
+1 …I've heard this a few times too
